src/main/java/com/synopsys/integration/detect/lifecycle/run/step/ContainerScanStepRunner.java

```diff
@@ -148,6 +164,36 @@ private void uploadImageToStorageService() throws IOException, IntegrationExcept
             }
         }
     }
+    
+    private DefaultUploadStatus multiPartUploadImage() throws IntegrationException {
+        String storageServiceEndpoint = String.join("", STORAGE_CONTAINERS_ENDPOINT, scanId.toString());
+        ContainerUploader containerUploader = uploadFactory.createContainerUploader(storageServiceEndpoint);
+        
+        DefaultUploadStatus status;
+        
+        try {
+            logger.debug("Performing multipart container image upload to storage endpoint: {}", storageServiceEndpoint);
+            status = containerUploader.upload(containerImage.toPath());
+        } catch (IOException | IntegrationException e) {
+            logger.trace("Unable to upload multipart container image.");
+            throw new IntegrationException("Unable to upload multipart container image.", e);
+        }
+            
+        if (status != null && status.isError()) {
```

Comment:
Can the status indeed be null here? If so, should we handle that somehow separately?

Reply:
Thanks, added a specific case for the null instead of continuing.